content/english/java-datastructures/trees/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Introduction"
+date: 2022-08-06T13:24:17-07:00
+draft: false
+weight: 1
+--- 
+
+Trees are made up of two components: **edges** and **nodes**. Let's take a look at each of these components in a bit more detail.
+
+### Edges
+
+You can imagine an edge as the line that connects two nodes. Edges always connect exactly two nodes. Edges can carry some information, but they don't need to.
+
+### Nodes
+
+Nodes are just points on a tree that represent some data. A tree needs at least one node and each node can have as many edges as it wants, as long as there is another different node that it is connected to.
+
+### Why Trees?
+
+Trees are very simple data structures that can store a lot of data while still allowing you to find it quickly. Many algorithms that rely on finding the best match organize their data in a tree before searching to optimize their speed. Some examples of trees being used in the real world include:
+* [Spanning Tree Protocol](https://en.wikipedia.org/wiki/Spanning_Tree_Protocol) - A protocol foundational to using the Internet, 
```

Comment:
Can you use <a href="..." target="_blank>...</a> instead of the markdown links so that these links open in a new window?